sbncode/FlashMatch/template_generators/generate_simple_weighted_template.py

```diff
@@ -562,23 +586,23 @@ def generator(nuslice_tree, rootfile, pset):
         m.draw_3D(canv)
```

Comment:
It doesn't allow me to put this comment where it's relevant, but you need to update the path name of `draw_metrics`. Currently it's rewriting the plots.